olive/passes/onnx/inc_quantization.py

```diff
@@ -55,6 +55,11 @@
             If users set domain as auto, automatic detection for domain will be executed.
         """,
     ),
+    "workspace": PassConfigParam(
+        type_=str,
+        default_value=None,
+        description="""Workspace for Intel® Neural Compressor quantization.""",
```

Comment:
Could you help update the description to tell the details like what they do: 
https://github.com/intel/neural-compressor/blob/dfd083df3234aa27548391c381d3ac9dc8139676/neural_compressor/config.py#L189C25-L191C112

Reply:
Done